config/localization/en/app.json

```diff
@@ -260,7 +260,7 @@
       "WITHDRAW": "Withdraw",
       "YOU": "You"
     },
-    "ASSETS": {
+    "__ASSETS": {
```

Comment:
could we keep the stringKey generation under `ASSETS`? We will use the `generated` to type the dynamic asset descriptions to a `keyof STRING_KEYS` type.